src/ex12_js_dom/js/task-02.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+const menuItems = [{linkURL: '#', linkTxt: 'Settings'}, {linkURL: '#', linkTxt: 'Sign out'}];
+let menuBody;
+let menuLink;
+let clickScreen;
+let isShown = false;
+
+document.querySelector('.user-menu__btn').addEventListener('click', toggleMenu);
+function toggleMenu() {
+  if (!isShown){
+  document.querySelector('.user-menu__arrow').style.transform = 'rotateZ(180deg)';
+  menuBody = document.createElement('nav');
```

Comment:
а почему ты объявил переменные, которые используются только в функции, в начале скрипта?

Reply:
Because they store data that is used multiple times by different invocation of the function. When eventListener will call the function second time, the variables (that must store the elements to be removed from DOM) will be redeclared (.i.e, undefined).